content/_data/changelogs/weekly.yml

```diff
@@ -26652,6 +26652,7 @@
         message: |-
           Jenkins' own user database no longer accepts new passwords longer than supported by bcrypt (72 bytes).
           Users with longer passwords are advised to change their password.
+          If you are using the Active Directory plugin, make sure to update to version 2.40 at the same time as updating Jenkins.
```

Comment:
The location of this line is confusing, as these are separate, independent security realms.

Reply:
I think we should have a new entry in the changelog for the removal, using the line from the pull request:

```
Remove jbcrypt library. If you're using Active Directory plugin, make sure to update to version 2.40 at the same time as updating Jenkins.
```

I made the mistake of not including that as a separate entry when I was experimenting with the split of the changelog entries.

Reply:
PR for entry: https://github.com/jenkins-infra/jenkins.io/pull/8094